dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/LibertyCXFNegativePropertiesTest.java

```diff
@@ -129,8 +129,8 @@ public void testCxfAttachmentOutputProperty() throws Exception {
     }
 
     private void connect(String methodName, int ExpectedConnection) throws Exception {
-        URL url = new URL("https://" + server.getHostname() + ":" + server.getHttpDefaultSecurePort()
-                          + "/webServiceRefFeatures/wsapolicyskip?impl=" + methodName);
+        String serverURL = "https://" + server.getHostname() + ":" + server.getHttpDefaultSecurePort();
+        URL url = new URL(serverURL + "/webServiceRefFeatures/wsapolicyskip?impl=" + methodName + "&serverurl=" + serverURL);
         Log.info(c, "LibertyCXFPositivePropertiesTest",
```

Comment:
Since you're here: can you change `LibertyCXFPositivePropertiesTest` to `LibertyCXFNegativePropertiesTest` in this log statement.